src/Catalog/CommonUtility.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class CommonUtility
```

Comment:
I think you might be able to get away with calling this either `Common` or `Utility` instead of the entire `CommonUtility`